public/signupLogin/signupLoginController.js

```diff
@@ -0,0 +1,44 @@
+(function() {
+  angular.module('app.signupLogin', [])
+    .controller('signupLoginController', signupLoginController)
+
+  signupLoginController.$inject = ['$window', '$location', 'Auth'];
+
+  function signupLoginController($window, $location, Auth) {
+    
+      var vm = this;
+      vm.signup = signup;
+      vm.signin = signin;
+      vm.signout = Auth.signout;
+
+    function signup(user) {
+      Auth.signup(user)
+        .then(function(token) {
```

Comment:
anonymous function! I would recommend moving to a named function within the scope of signup()